internal/pkg/acl/acl.go

```diff
@@ -317,41 +314,27 @@ func CreateAclRequestDataToAclData(data *AclRequestDataWithError) cpkafkarestv3.
 	}
 }
 
-func PrintACLsFromKafkaRestResponseWithResourceIdMap(cmd *cobra.Command, acls []cckafkarestv3.AclData, numericIdToResourceId map[int32]string) error {
+func PrintACLsFromKafkaRestResponse(cmd *cobra.Command, acls []cckafkarestv3.AclData) error {
 	list := output.NewList(cmd)
 	for _, acl := range acls {
-		prefix, resourceId, err := getPrefixAndResourceIdFromPrincipal(acl.Principal, numericIdToResourceId)
-		if err != nil {
-			if err.Error() == errors.UserIdNotValidErrorMsg {
-				continue // skip the entry if not a valid user id
-			}
-			return err
-		}
 		list.Add(&out{
-			Principal:    prefix + ":" + resourceId,
-			Permission:   acl.Permission,
-			Operation:    acl.Operation,
-			ResourceType: string(acl.ResourceType),
-			ResourceName: acl.ResourceName,
-			PatternType:  acl.PatternType,
+			Principal:    acl.GetPrincipal(),
+			Permission:   acl.GetPermission(),
+			Operation:    acl.GetOperation(),
+			ResourceType: string(acl.GetResourceType()),
+			ResourceName: acl.GetResourceName(),
+			PatternType:  acl.GetPatternType(),
 		})
 	}
 	list.Filter(listFields)
 	return list.Print()
 }
 
-func PrintACLsWithResourceIdMap(cmd *cobra.Command, acls []*ccstructs.ACLBinding, numericIdToResourceId map[int32]string) error {
+func PrintACLsWithResourceIdMap(cmd *cobra.Command, acls []*ccstructs.ACLBinding) error {
```

Comment:
We can probably cut `WithResourceIdMap` from this function name.

Reply:
Thanks, looks like this function was a duplicate!